Tests/JournalMonitorTests.cs

```diff
@@ -473,15 +473,16 @@ public void TestJournalMissionAccepted1()
         [TestMethod]
         public void TestFriends()
         {
-            string line = "{ \"timestamp\":\"2017-08-24T17:22:03Z\", \"event\":\"Friends\", \"Status\":\"Online\", \"Name\":\"_Testy_McTest_\" }";
+            string line1 = "{ \"timestamp\":\"2017-08-24T17:22:03Z\", \"event\":\"Friends\", \"Status\":\"Online\", \"Name\":\"_Testy_McTest_\" }";
+            string line2 = "{ \"timestamp\":\"2017-08-24T17:22:03Z\", \"event\":\"Friends\", \"Status\":\"Offline\", \"Name\":\"_Testy_McTest_\" }";
 
-            List<Event> events = JournalMonitor.ParseJournalEntry(line);
+            List<Event> events1 = JournalMonitor.ParseJournalEntry(line1);
+            List<Event> events2 = JournalMonitor.ParseJournalEntry(line2);
 
-            /// Since this friend is unknown to us, the first time we see this friend no event should trigger. 
-            /// Only the second line, registering the status as offline, should be registered as an event.
```

Comment:
See the `eventFriends` method in EDDI.cs for context on what this test was (very poorly) trying to test. 
The test should be modified to call the `eventFriends` method. It should be false for line1 and true for line2.